common/bitvec/src/lib.rs

```diff
@@ -31,15 +31,14 @@ const MAX_BUCKETS: usize = 32;
 /// * A position can set more than once -- it remains set after the first time.
 ///
 /// # Examples:
-/// ```ignore
```

Comment:
Deleting this line will disable to doctest, so let's keep it and change the `ignore` to `rust` so our example doesn't drift : )